cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2318,10 +2318,8 @@ action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Office 365
 action.cyences_notable_event_action.teams = SOC
 
-# Possible user_type field values for the O365 management activity
-# https://learn.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema#user-type
 
-[O365 - Risky Login Detected by Microsoft]
```

Comment:
This is alert deleted, so need to write upgrade steps for this. Release notes. Upgrade guide.